### ISSUE TITLE
Question on determining whether aria-labelledby maps to AXDescription or AXTitle

### ISSUE THREAD

**Issue by sivakusayan** [w3c/core-aam#213]
The [current mapping](https://www.w3.org/TR/core-aam-1.2/#ariaLabelledBy) states that browsers should expose `aria-labelledby` through `AXTitle` if the value is rendered visually. However, if I'm reading [WebKit's AXDescription code](https://github.com/WebKit/WebKit/blob/main/Source/WebCore/accessibility/AccessibilityNodeObject.cpp#L2105) right, it will always expose `aria-labelledby` via `AXDescription`. Very briefly going over [this WPT page](https://wpt.fyi/results/accname/name/comp_labelledby_hidden_nodes.html?label=master&label=experimental&aligned) with the Mac accessibility inspector seems to confirm my hypothesis (for buttons, anyway). [Webkit's AXTitle code](https://github.com/WebKit/WebKit/blob/main/Source/WebCore/accessibility/AccessibilityNodeObject.cpp#L2397) seems to be mostly used to expose name from content and HTML labels.

I think it would be nice to get some clarification of whether we should be following WebKit or the spec here - this discrepancy makes me unsure how to fix [this Chromium bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1396300&q=component%3ABlink%3EAccessibility&can=2&start=100) where a dialog's title isn't read when focus moves into the dialog. This happens because VoiceOver (tested on Ventura 13.5) won't respect a dialog's `AXTitle` if `AXTitleUIElement` is null, but I'm not sure if we can guarantee that `AXTitleUIElement` always exists (for example, if `aria-labelledby` points to two nodes). Safari gets past this by exposing the dialog's title through `AXDescription`.

**Comment by cookiecrook** [comment]
@twilco can you review if we have a WebKit bug or if this needs a docs change? Thanks.
